sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/EntityHelperTest.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.data.tables.models.TableEntity;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class EntityHelperTest {
+    private final ClientLogger logger = new ClientLogger(EntityHelperTest.class);
+
+    @Test
+    public void testConvertToSubclass() {
```

Comment:
Something new in JUnit 5 is that test methods no longer have to be public. So this class could be package-private and same with its methods.

Reply:
Interesting, I guess there's really no point to visibility modifiers when the class just contains tests.